scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -195,11 +195,11 @@ def update_custom_gates_navigation_url(plKeyExecDict):
                 exec_str = str(execution.decode("utf-8"))
                 field = "stage." + exec_str + ".outputs"
                 type = "stage."+exec_str+".type"
-                print("The pl key is: ", plKey)
-                print("The field is: ", field)
+                logging.info("The pl key is: %s", plKey)
+                logging.info("The field is: %s", field)
```

Comment:
@sudhakaropsmx I don't this this syntax works, don't even see these messages in the migration log from last successful run.
What we saw previously, it should be in this format - 
logging.info("The pl key is: " + str(plKey))

Reply:
or may be it worked but added after running the scritp so not present in log ? please confirm...

Reply:
will work no changes required